nixos/modules/hardware/ledger-nano-s.nix

```
@@ -0,0 +1,32 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.hardware.ledger-nano-s;
+
+in
+
+{
+  options.hardware.ledger-nano-s = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enables udev rules for Ledger Nano S devices (https://www.ledger.com/products/ledger-nano-s).
```

Would be nice for the link to be a real link:
```suggestion
        Enables udev rules for Ledger Nano S devices
        (<link xlink:href="https://www.ledger.com/products/ledger-nano-s"/>).
```